assets/sass/components/analytics-4/_googlesitekit-analytics-4-reminder-banner.scss

```diff
@@ -50,12 +50,20 @@
 
 		.googlesitekit-publisher-win__secondary-pane {
 
-			ul {
-				list-style-image: $icon-tick;
+			.googlesitekit-publisher-win__secondary-pane-title {
+				font-size: $fs-title-sm;
+				font-weight: $fw-medium;
+				line-height: 0;
```

Comment:
We should not be setting a `line-height: 0;` because it will lead to issues when the text spans multiple lines:

<img width="1102" alt="CleanShot 2023-03-22 at 20 28 07" src="https://user-images.githubusercontent.com/90871/227030139-4c95eb78-81d7-4f16-8480-96e18b20072d.png">

What is the reason for this style? If it's to match the Figma design, something like:

```suggestion
				margin: 8px 0 0;
```

Along with a `margin-top` of `8px` for the `ul` would work.